Source/SmashBros/Game/Characters/Cuphead.h

```diff
@@ -0,0 +1,100 @@
+
+#include "../../Player.h"
+
+#pragma once
+
+namespace SmashBros
+{
+	class Fox : public Player
+	{
```

Comment:
looks like you may have forgotten to rename this spot ^

Reply:
I took Fox as a base, so thanks!